src/main/java/sx/blah/discord/api/internal/DiscordVoiceWS.java

```diff
@@ -257,11 +257,13 @@ public void run() {
 	 * Sets up the thread for receiving audio packets from the Discord UDP Socket.
 	 */
 	private void setupReceiveThread() {
-		Runnable receiveThread = () -> {
+		(new Thread((Runnable) () -> {
+                    while(true)
```

Comment:
Should only continue while the udp socket is open / audio is connected and the thread isn't interrupted.